app/lib/server/methods/addUsersToRoom.js

```diff
@@ -65,7 +68,22 @@ Meteor.methods({
 					method: 'addUsersToRoom',
 				});
 			}
-			addUserToRoom(data.rid, newUser, user);
+			// addUserToRoom(data.rid, newUser, user);
```

Comment:
@gsunit Remove this commented code, please.

Reply:
I see that you have opened a new PR. Thanks! Closing this one then.